Run/navstart.ps1

```diff
@@ -251,13 +257,15 @@ if ($runningGenericImage -or $runningSpecificImage -or $buildingImage) {
     $CustomConfig.Save($CustomConfigFile)
 }
 
-if ($runningGenericImage -or $runningSpecificImage) {
+if ($runningGenericImage -or $runningSpecificImage -or $restartingInstance) {
```

Comment:
I already commented this with @freddydk and I proposed to disable the SQL instance directly in the case you are using an external SQL service. You don\`t need to handle it in case of restarts. @freddyk stated that instead of disabling it directly he would change the startup type to **manual**.